day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,148 @@
+#10 speckled frogs sat on a log 
+#eating some most delicious bugs.  
+#One jumped in the pool where its nice and cool,
+#then there were 2 speckled frogs.
+###required###
+
+#did some googling and found out that .step exists, which is infinitely more convenient than what I had previously tried to do.
+frogs = 10
+
+frogs.step(1, -1) do |x|

```

Comment:
Cool that you found `step` as an iterator. More commonly you might see `times` like `frogs.times do |x|`.